src/table_row.rs

```diff
@@ -244,9 +219,7 @@ fn get_renderer_for_field(name: &Ident, field: &TableRowField, index: usize) ->
             )
         }
     } else {
-        quote! {
-            <leptos_struct_table::DefaultTableCellRenderer #format_props #value_prop #class_prop on_change=|_| {} />
-        }
+        panic!("This is not supported")
```

Comment:
When can this even occur? We cant figure out the type in this case so i made it panic... unsure what to do here.

Reply:
Hm I think you're right. I guess we'll find out 😀